bytecomp/lambda.ml

```diff
@@ -103,6 +103,7 @@ type primitive =
   | Pbigarrayset of bool * int * bigarray_kind * bigarray_layout
   (* size of the nth dimension of a big array *)
   | Pbigarraydim of int
+  | Pbigarraybytesize
```

Comment:
Why add this as a lambda primitive?  It's just an external C function like any other, without special compiler treatment, no?